README.md

```diff
@@ -1,3 +1,12 @@
+## KNOT Android library
+
+1. This lib is a part of KNOT solution (for more information: https://github.com/CESARBR) and has aims to provide abstractions for Android application create owner IoT (Internet of Things) solution in KNOT platform by HTTP protocol and via SocketIO.
+
+## knot-lib-android-source Goals
+1. Provide abstraction to connect with KNOT cloud by HTTP or SocketIO, it is possible to cofigure with specific cloud, CRUD of devices, send messages and others.
```

Comment:
It should read

    1. Provide abstraction to connect with KNOT cloud using HTTP or Sockets. It is possible to connect with a specific cloud, perform CRUD operations in devices, send messages, and more.